notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+{
```

Comment:
<p>Is it worth linking this <a href="https://bugzilla.redhat.com/docs/en/html/using/understanding.html" target="_blank">understanding a bug</a> documentation here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Yes I think. But this document does not contain information for all the fields :( </p>

Reply:
<p>Yeah we had been using a combination of the <a href="https://github.com/python-bugzilla/python-bugzilla/blob/82972796cf04f1ac06525670c272465a66d77da1/man/bugzilla.rst#global-options" target="_blank">python bugzilla library docs</a> and <a href="https://bugzilla.readthedocs.io/en/latest/using/understanding.html" target="_blank">bugzilla docs</a> (which I think is essentially the same doc you mentioned) to learn about the fields. So definitely worth it to put all these links here for the curious reader.</p>